diff --git a/arch/arm/boot/dts/imx6qdl-var-solocb-dt6cb-displays.dtsi b/arch/arm/boot/dts/imx6qdl-var-solocb-dt6cb-displays.dtsi
index 45e5c95..485fd41 100644
--- a/arch/arm/boot/dts/imx6qdl-var-solocb-dt6cb-displays.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-var-solocb-dt6cb-displays.dtsi
@@ -30,37 +30,37 @@
 		display-timings {
 			native-mode = <&timing0>;
 			timing0: hsd100pxn1 {
-				clock-frequency = <35714000>;
-				hactive = <800>;
-				vactive = <480>;
-				hback-porch = <24>;
-				hfront-porch = <15>;
-				vback-porch = <13>;
-				vfront-porch = <20>;
-				hsync-len = <20>;
-				vsync-len = <13>;
+				clock-frequency = <44426000>;
+				hactive = <1152>;
+				vactive = <576>;
+				hback-porch = <32>;
+				hfront-porch = <40>;
+				vback-porch = <14>;
+				vfront-porch = <3>;
+				hsync-len = <8>;
+				vsync-len = <8>;
 			};
 		};
 	};
 
 	lvds-channel@1 {
 		fsl,data-mapping = "spwg";
-		fsl,data-width = <18>;
+		fsl,data-width = <24>;
 		primary;
 		status = "okay";
 
 		display-timings {
 			native-mode = <&timing1>;
 			timing1: hsd100pxn1 {
-				clock-frequency = <32000000>;
-				hactive = <800>;
-				vactive = <480>;
-				hback-porch = <39>;
-				hfront-porch = <39>;
-				vback-porch = <29>;
-				vfront-porch = <13>;
-				hsync-len = <47>;
-				vsync-len = <2>;
+				clock-frequency = <44426000>;
+				hactive = <1152>;
+				vactive = <576>;
+				hback-porch = <32>;
+				hfront-porch = <40>;
+				vback-porch = <14>;
+				vfront-porch = <3>;
+				hsync-len = <8>;
+				vsync-len = <8>;
 			};
 		};
 	};
